src/api/ApiClient.js

```diff
@@ -3,16 +3,44 @@ import base64 from 'base-64';
 
 const apiVersion = 'api/v1';
 
+export type Account = {
+  accountId: number,
+  activeBackend: 'dev' | 'google' | 'password',
+  apiKey: string,
+  authBackends: string[],
+  email: string,
+  directAdmins: string[],
```

Comment:
`directAdmins` and `directUsers` are optional -- I assume that needs to be indicated in the Flow type?